internal/cmd/manager/instance/run/cmd.go

```diff
@@ -115,6 +116,8 @@ func runSubCommand(ctx context.Context, instance *postgres.Instance) error {
 		"version", versions.Version,
 		"build", versions.Info)
 
+	gracefulShutdownTimeout := time.Duration(int64(instance.SmartStopDelay) * int64(time.Second))
```

Comment:
I recommend using `instance.MaxStopDelay` instead of `instance.SmartStopDelay`, as the latter is solely for smart shutdowns. When a smart shutdown exceeds its timeout limit, it automatically switches to a fast shutdown. However, if the graceful shutdown of the instance also times out, it results in the same problem